aot/src/main/java/com/dylibso/chicory/aot/AotMethodRefs.java

```diff
@@ -60,7 +60,7 @@ public final class AotMethodRefs {
             INSTANCE_MEMORY = Instance.class.getMethod("memory");
             INSTANCE_CALL_HOST_FUNCTION =
                     Instance.class.getMethod("callHostFunction", int.class, long[].class);
-            INSTANCE_READ_GLOBAL = Instance.class.getMethod("readGlobal", int.class);
+            READ_GLOBAL = AotMethods.class.getMethod("readGlobal", int.class, Instance.class);
```

Comment:
I'd say this and related are the most notable changes in the PR. The rest of it is changes to the visibility of some class and methods so that they are all package-private